cmd/policy_rm.go

```diff
@@ -23,9 +23,12 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const allKeyword = "all"
+
 func newPolicyRmCmd() *cobra.Command {
+
 	var cmd = &cobra.Command{
-		Use:   "rm <org-name>/<policy-pack-name> <version>",
+		Use:   "rm <org-name>/<policy-pack-name> <all|version>",
```

Comment:
I thought the plan was to align with `policy_enable.go` and either have to specify a `version` arg or specify a `--all` flag (like `enable` having to either specify a version or `--latest`)?

Reply:
Yes it was! But then when I am implemented it I found the usage a bit strange and confusing. Because `version` would be listed as optional, but it really was only optional if there was no `-all` flag. Happy to change it back but I thought this made it a bit more straighforward

Reply:
Makes sense. Should we change `enable` to use `latest` rather than `--latest`? Otherwise, it feels weird to me that we're using two different patterns between these two commands.

Reply:
sounds good will update that in this PR!

Reply:
fixed plz take a look!